selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/chat/ChatStatus.java

```diff
@@ -0,0 +1,18 @@
+package org.exoplatform.platform.qa.ui.selenium.platform.chat;
+
+
+import org.exoplatform.platform.qa.ui.selenium.TestBase;
+import org.exoplatform.platform.qa.ui.selenium.testbase.ElementEventTestBase;
+
+public class ChatStatus  {
+	private final TestBase testBase;
+	private ElementEventTestBase evt;
+
+    public ChatStatus(TestBase testBase) {
```

Comment:
Please format the code with eXo Formatter Settings http://developer.exoplatform.org/#id-formatter-settings